parm/soca/obs/config/adt_j3.yaml

```diff
@@ -0,0 +1,11 @@
+obs space:
+  name: adt_j3
+  obsdatain:
+    obsfile: $(experiment_dir)/{{current_cycle}}/adt_j3.{{window_begin}}.nc4
```

Comment:
FYI I'm not following the 'ewok' convention for these paths/filenames, suggest you don't do so either. But it's up to you.

Reply:
I'll keep this for now a change when I actually do something with these yaml's.